runtime/compiler/control/rossa.cpp

```diff
@@ -1923,6 +1923,14 @@ onLoadInternal(
 
    jitConfig->jitAddNewLowToHighRSSRegion = jitAddNewLowToHighRSSRegion;
 
+#ifndef J9VM_OPT_JITSERVER
+   if (TR::Options::getCmdLineOptions()->getTransientClassRegex())
+      {
+      compInfo->setTransientClassLoadersSet(new (PERSISTENT_NEW) PersistentUnorderedSet<J9ClassLoader*>(PersistentUnorderedSet<J9ClassLoader*>::allocator_type(TR::Compiler->persistentAllocator())));
+      _transientClassLoaderMonitor = TR::Monitor::create("JIT-TransientClassLoadersMonitor");
```

Comment:
We can fail the JVM if the monitor cannot be created. Thus, we can avoid testing for the existence of the monitor in other parts of the code.